content/intro_to_rbac/create_iam_user.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Create a User"
+date: 2018-10-03T10:14:46-07:00
+draft: false
+weight: 20
+---
+
+{{% notice note %}}
+For the sake of simplicity, in this chapter, we will save credentials to a file to make it easy to toggle back and forth between users.  Never do this in production or with credentials that have priveledged access; It is not a security best practice to store credentials on the filesystem.
+{{% /notice %}}
+
+From within the Cloud9 terminal, create a new user called rbac-user, and generate credentials for it:
+
+{{< tabs name="Create EKS Cluster" >}}
+
+{{{< tab name="Workshop at AWS event"  >}}
+
+From within the Cloud9 terminal, run the script given to you by the AWS Workshop Staff.
```

Comment:
I'm not sure why you are saying this...

Reply:
@dalbhanj we're going to modify the CF template to auto-create this user. In the meantime, due to permissions settings, its not allowed to create a user, so we provide the students with a script that has creds for a user pre-created just for the workshop to use instead.